### ISSUE TITLE
Invalid type specified "datetime" usin MongoDB

### ISSUE THREAD

**Issue by Cypaubr** [markitosgv/JWTRefreshTokenBundle#175]
I got this error while login:
```
InvalidArgumentException:
Invalid type specified "datetime".

  at vendor/doctrine/mongodb-odm/lib/Doctrine/ODM/MongoDB/Types/Type.php:137
  at Doctrine\ODM\MongoDB\Types\Type::getType('datetime')
     (vendor/doctrine/mongodb-odm/lib/Doctrine/ODM/MongoDB/Persisters/PersistenceBuilder.php:81)
  at Doctrine\ODM\MongoDB\Persisters\PersistenceBuilder->prepareInsertData(object(RefreshToken))
     (vendor/doctrine/mongodb-odm/lib/Doctrine/ODM/MongoDB/Persisters/DocumentPersister.php:208)
  at Doctrine\ODM\MongoDB\Persisters\DocumentPersister->executeInserts(array())
     (vendor/doctrine/mongodb-odm/lib/Doctrine/ODM/MongoDB/UnitOfWork.php:1062)
  at Doctrine\ODM\MongoDB\UnitOfWork->executeInserts(object(ClassMetadata), array('000000002c1524ac000000000456911d' => object(RefreshToken)), array())
     (vendor/doctrine/mongodb-odm/lib/Doctrine/ODM/MongoDB/UnitOfWork.php:410)
  at Doctrine\ODM\MongoDB\UnitOfWork->commit(array())
     (vendor/doctrine/mongodb-odm/lib/Doctrine/ODM/MongoDB/DocumentManager.php:562)
  at Doctrine\ODM\MongoDB\DocumentManager->flush()
     (vendor/gesdinet/jwt-refresh-token-bundle/Doctrine/RefreshTokenManager.php:79)
  at Gesdinet\JWTRefreshTokenBundle\Doctrine\RefreshTokenManager->save(object(RefreshToken))
     (vendor/gesdinet/jwt-refresh-token-bundle/EventListener/AttachRefreshTokenOnSuccessListener.php:139)
  at Gesdinet\JWTRefreshTokenBundle\EventListener\AttachRefreshTokenOnSuccessListener->attachRefreshToken(object(AuthenticationSuccessEvent), 'lexik_jwt_authentication.on_authentication_success', object(TraceableEventDispatcher))
     (vendor/symfony/event-dispatcher/Debug/WrappedListener.php:126)
  at Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(object(AuthenticationSuccessEvent), 'lexik_jwt_authentication.on_authentication_success', object(TraceableEventDispatcher))
     (vendor/symfony/event-dispatcher/EventDispatcher.php:247)
  at Symfony\Component\EventDispatcher\EventDispatcher->callListeners(array(object(WrappedListener)), 'lexik_jwt_authentication.on_authentication_success', object(AuthenticationSuccessEvent))
     (vendor/symfony/event-dispatcher/EventDispatcher.php:73)
  at Symfony\Component\EventDispatcher\EventDispatcher->dispatch(object(AuthenticationSuccessEvent), 'lexik_jwt_authentication.on_authentication_success')
     (vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php:168)
  at Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(object(AuthenticationSuccessEvent), 'lexik_jwt_authentication.on_authentication_success')
     (vendor/lexik/jwt-authentication-bundle/Security/Http/Authentication/AuthenticationSuccessHandler.php:50)
  at Lexik\Bundle\JWTAuthenticationBundle\Security\Http\Authentication\AuthenticationSuccessHandler->handleAuthenticationSuccess(object(User))
     (vendor/lexik/jwt-authentication-bundle/Security/Http/Authentication/AuthenticationSuccessHandler.php:37)
  at Lexik\Bundle\JWTAuthenticationBundle\Security\Http\Authentication\AuthenticationSuccessHandler->onAuthenticationSuccess(object(Request), object(UsernamePasswordToken))
     (vendor/symfony/security-http/Authentication/CustomAuthenticationSuccessHandler.php:45)
  at Symfony\Component\Security\Http\Authentication\CustomAuthenticationSuccessHandler->onAuthenticationSuccess(object(Request), object(UsernamePasswordToken))
     (vendor/symfony/security-http/Firewall/AbstractAuthenticationListener.php:211)
  at Symfony\Component\Security\Http\Firewall\AbstractAuthenticationListener->onSuccess(object(Request), object(UsernamePasswordToken))
     (vendor/symfony/security-http/Firewall/AbstractAuthenticationListener.php:138)
  at Symfony\Component\Security\Http\Firewall\AbstractAuthenticationListener->__invoke(object(RequestEvent))
     (vendor/symfony/security-bundle/Debug/WrappedListener.php:51)
  at Symfony\Bundle\SecurityBundle\Debug\WrappedListener->__invoke(object(RequestEvent))
     (vendor/symfony/security-bundle/Debug/TraceableFirewallListener.php:35)
  at Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener->callListeners(object(RequestEvent), object(Generator))
     (vendor/symfony/security-http/Firewall.php:97)
  at Symfony\Component\Security\Http\Firewall->onKernelRequest(object(RequestEvent), 'kernel.request', object(TraceableEventDispatcher))
     (vendor/symfony/event-dispatcher/Debug/WrappedListener.php:126)
  at Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(object(RequestEvent), 'kernel.request', object(TraceableEventDispatcher))
     (vendor/symfony/event-dispatcher/EventDispatcher.php:260)
  at Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(array(object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener)), 'kernel.request', object(RequestEvent))
     (vendor/symfony/event-dispatcher/EventDispatcher.php:235)
  at Symfony\Component\EventDispatcher\EventDispatcher->callListeners(array(object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener)), 'kernel.request', object(RequestEvent))
     (vendor/symfony/event-dispatcher/EventDispatcher.php:73)
  at Symfony\Component\EventDispatcher\EventDispatcher->dispatch(object(RequestEvent), 'kernel.request')
     (vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php:168)
  at Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(object(RequestEvent), 'kernel.request')
     (vendor/symfony/http-kernel/HttpKernel.php:127)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/http-kernel/HttpKernel.php:68)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/http-kernel/Kernel.php:198)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (public/index.php:25)
```

**Comment by lcavero** [comment]
@markitosgv Could you fix that please? It's easy, I think. Just change datetime type to date.

**Comment by fanch317** [comment]
Work in previous version 0.8.1

@lcavero You can propose a MR with this fix or use version 0.8.1.

**Comment by markitosgv** [comment]
Fixed in e3b2db2563982629a77ed4521df60a0e8ac2df64 and tagged as v0.8.3
